CodingStandards/ProgramEntryPoint.cs

```diff
@@ -4,34 +4,50 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
-
+/// <summary>
+/// Creates Program class to get current date and time to display appropriate message.
+/// </summary>
 class Program
```

Comment:
It looks like you based this assignment on a branch other than master again. Ensure that you start with master for your coding exercises.